feat: Added ui feedback when adding type in typesync

unniznd opened the pull request:
UI Feedback when you added a new type from the left panel
Closes #498

cmwhited reviewed (changes requested):
Appreciate you taking this task on!
Few things:
1. make sure to format and lint your code
2. I am not sure the typeRefs map and useEffect is the best way to scroll the newly added type into view. could just add the scroll into view in the `KnowledgeGraphBrowser.typeSelected` instead.  much less state to manage and the `useEffect` has me really worried

unniznd commented:
@cmwhited thank you for your feedback. I will work on it.

unniznd commented:
@cmwhited 
I have removed the usage of useEffect and added it in ```KnowledgeGraphBrowser.typeSelected``` itself. Can you help me with linting the code? The ```pnpm lint:fix``` is giving me errors.

cmwhited commented:
> @cmwhited I have removed the usage of useEffect and added in `KnowledgeGraphBrowser.typeSelected` the scrolling logic. Can you help me with linting the code? The `pnpm lint:fix` is giving me errors.

what errors is it giving you?

unniznd commented:
I have fixed the issue in linting. I was having a package installation issue. I have updated the code. Can you review and give some feedback @cmwhited

unniznd commented:
@cmwhited, I have updated the code. Can you review and give some feedback?
